Server/tests/test_resource_uri_references.py

```
@@ -0,0 +1,143 @@
+        offenders += _offenders(
+            tool.get("kwargs", {}).get("description"),
+            f"tool '{name}' description",
+            resource_uris_by_name,
+        )
+        offenders += _offenders(
+            func.__doc__, f"tool '{name}' docstring", resource_uris_by_name)
+
+        try:
+            hints = typing.get_type_hints(func, include_extras=True)
+        except Exception:
+            continue
```

Ruff (0.16.0)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)